Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM opensciencegrid/software-base:fresh
+# Specify the opensciencegrid/software-base image tag
+ARG SW_BASE_TAG=fresh
```

Comment:
I'll be honest, i don't understand the interaction between this `ARG SW_BASE_TAG=fresh` in `Dockerfile` and the `build_args: SW_BASE_TAG=${{ matrix.dver }}fresh` line in the github workflow yaml.

Does the `SW_BASE_TAG` set in the workflow override the value set in the `ARG` line in the Dockerfile?

Reply:
Yup! This line in the Dockerfile sets the default `SW_BASE_TAG` to `fresh`, which is useful for our concurrent Travis builds (until we turn them off) and folks doing manual builds.

Reply:
Ok thanks for the info!